e2e/config/protractor.conf.ts

```diff
@@ -0,0 +1,157 @@
+// Because this file imports from  protractor, you'll need to have it as a
+// project dependency. Please see the reference config: lib/config.ts for more
+// information.
+//
+// Why you might want to create your config with typescript:
+// Editors like Microsoft Visual Studio Code will have autocomplete and
+// description hints.
+//
+// To run this example, first transpile it to javascript with `npm run tsc`,
+// then run `protractor conf.js`.
+import * as protractor from 'protractor';
+import * as puppeteer from 'puppeteer';
+import * as reporter from 'multiple-cucumber-html-reporter';
+import * as path from 'path';
+import * as fs from 'fs';
+import * as mkdirp from 'mkdirp';
+import * as moment from 'moment';
+import * as e2econfig from './e2e.conf.json';
+import { browser } from 'protractor';
+import undefinedError = Mocha.utils.undefinedError;
+
+const jsonReports = path.join(process.cwd(), 'e2e/reports/json');
+const htmlReports = path.join(process.cwd(), 'e2e/reports/html');
+const screenshots = path.join(process.cwd(), 'e2e/reports/screenshots');
+let scenario_tags = [];
+scenario_tags.push("~@wip");
+
+const setupDeviceEmulation = () => {
+  const tags          = process.env.PLATFORM_TAG || "@desktop";
+  const isMobile      = tags.includes('@mobile');
+  const isTablet      = tags.includes('@tablet');
+  const isDesktop     = tags.includes('@desktop');
+
+  if (isMobile) {
+    return {
+      'deviceName': 'iPhone 7'
+    }
+  } else if (isTablet) {
+    return {
+      'deviceName': 'iPad'
+    }
+  } else if (isDesktop) {
+    return undefined;
+  }
+};
+
+const cucumberReporterOptions = {
+  //ignore: [],
+  jsonDir: jsonReports,
+  reportPath: htmlReports,
+  metadata: {
+    browser: {
+      name: 'chrome',
+      version: '60'
+    },
+    device: 'Local test machine',
+    platform: {
+      name: 'Windows'
+    }
+  },
+  customData: {
+    title: 'Run info',
+    data: [
+      { label: 'Project', value: 'Lott Protractor Tests' },
```

Comment:
Change this name to something else please, it is just a label in the test report